lib/cyclonedx/base.rb

```diff
@@ -33,7 +33,8 @@ def parse_dependency(dependency)
         "properties": [
           {
             "key": "source",
-            "value": dependency[:source]
+            # Varies between these two values by scanner
+            "value": dependency[:source] || dependency[:reference]
```

Comment:
Aside: We probably should consider standardizing the existing dependency reporters.  This would allow you to only look for source here.  Not something we need to hit in this PR - just making a note.